server/cmwell-util/src/main/scala/cmwell/util/http/SimpleHttpClient.scala

```diff
@@ -136,15 +136,36 @@ object SimpleHttpClient extends LazyLogging {
   }.toList
 
   private def mkURI(uri: String, queryParams: Seq[(String,String)]) = {
-    if (queryParams.isEmpty) uri
+
+    val noSchemeWithPort = uri.matches("\\w+:\\d+")
```

Comment:
\\w does include digits but not hyphen or dot and those won't be caught properly